pandas/tests/scalar/interval/test_arithmetic.py

```diff
@@ -0,0 +1,54 @@
+from datetime import timedelta
+
+from numpy import timedelta64
+import pytest
+
+from pandas import Interval, Timedelta, Timestamp
+
+
+@pytest.mark.parametrize("method", ["__add__", "__sub__"])
+@pytest.mark.parametrize(
+    "delta", [Timedelta(days=7), timedelta(7), timedelta64(7, "D")]
```

Comment:
this can probably use either `three days` or `scalar_td` from the arithmetic fixtures (would need to be moved up to base conftest)